### ISSUE TITLE
API call to set rate on the fly

### ISSUE THREAD

**Issue by iongion** [bikegriffith/clappr-playback-rate-plugin#2]
Hi, it is not an issue but more of a question. Let's say you have a player already playing, what calls do you need to perform to change the rate on-the-fly.

I have my player instance, but from there I do not know what and how to call so that the UI also updates properly such as becoming selected on 2x when 2x is chosen.

Something like:

```
player.setPlaybackRate(2); // will trigger 2x to be selected in the UI plus change the rate to 2
```

**Comment by bikegriffith** [comment]
The plugin does not currently support this.  Remember, this plugin only works for HTML5 playback (Clappr also supports Flash).

We could conceivably monkeypatch a method onto `player.core` when the plugin is initialized to do this.  Or you could work your way through the player.plugins to find this one and set it that way.  Less ugly might be to expose a new event that the plugin could listen to and you could trigger from the outside.  

Would you be willing to take a stab at a PR?

**Comment by bikegriffith** [comment]
Solved by v0.2.0
